src/main/resources/hudson/plugins/git/GitTagAction/tagForm.jelly

```diff
@@ -79,13 +82,17 @@ THE SOFTWARE.
             </div>
             <j:forEach var="m" items="${tags.keySet()}" varStatus="loop">
               <div class="tr">
-                <div><j:if test="${tags.size()!=1}">
-                  <input type="checkbox" name="tag${loop.index}" checked="true"
-                         id="tag${loop.index}" onchange="updateRow(this,${loop.index})" />
-                </j:if></div>
-                <div><label for="tag${loop.index}">
-                  ${m}
-                </label></div>
```

Comment:
Please also remove the script from the jelly template https://github.com/jenkinsci/git-plugin/blob/127ff25f77658937f88bce3d707fb4ee03731d99/src/main/resources/hudson/plugins/git/GitTagAction/tagForm.jelly#L105-L111